Automattic/wp-calypso
# Me: Localize apps download links on get-apps screen

#### Proposed Changes

* Localize apps download links on `/me/get-apps`
* Replace mobile app link QR code SVG with `qrcode.react` component to dynamically generate QR code for the localized mobile app page link

#### Testing Instructions

* Change UI to Mag-16 language
* Go to `/me/get-apps` and confirm all download app links and QR code link are localized

Related to 450-gh-Automattic/i18n-issues
